keyboards/binepad/bnr1/keymaps/binepad/config.h

```diff
@@ -0,0 +1,10 @@
+/* Copyright 2022 Binepad (@binpad) */
```

Comment:
File should be removed as it only contains VIAL defines, and those are not accepted to this repo.